packages/pkg-utils/lib/get-package.js

```diff
@@ -32,6 +32,11 @@ exports.getPackage = function getPackage(options) {
   return new Package(result.packageJson, path.dirname(result.path))
 }
 
+/**
+ * Reads a package.json
+ * @param options {readPkgUp.NormalizeOptions}
+ * @returns {readPkgUp.NormalizedPackageJson}
+ */
```

Comment:
new version of TS complains about not able to infer types if this comment is not here